src/autopas/options/ContainerOption.h

```diff
@@ -105,18 +105,16 @@ class ContainerOption : public Option<ContainerOption> {
    * @return map option -> string representation
    */
   static std::map<ContainerOption, std::string> getOptionNames() {
-    return {
-        {ContainerOption::directSum, "DirectSum"},
-        {ContainerOption::linkedCells, "LinkedCells"},
-        {ContainerOption::linkedCellsReferences, "LinkedCellsReferences"},
-        {ContainerOption::verletLists, "VerletLists"},
-        {ContainerOption::verletListsCells, "VerletListsCells"},
-        {ContainerOption::verletClusterLists, "VerletClusterLists"},
-        {ContainerOption::varVerletListsAsBuild, "VarVerletListsAsBuild"},
-        {ContainerOption::pairwiseVerletLists, "PairwiseVerletLists"},
-        {ContainerOption::octree, "Octree"},
-        {ContainerOption::hierarchicalGrids, "HierarchicalGrids"}
-    };
+    return {{ContainerOption::directSum, "DirectSum"},
+            {ContainerOption::linkedCells, "LinkedCells"},
+            {ContainerOption::linkedCellsReferences, "LinkedCellsReferences"},
+            {ContainerOption::verletLists, "VerletLists"},
+            {ContainerOption::verletListsCells, "VerletListsCells"},
+            {ContainerOption::verletClusterLists, "VerletClusterLists"},
+            {ContainerOption::varVerletListsAsBuild, "VarVerletListsAsBuild"},
+            {ContainerOption::pairwiseVerletLists, "PairwiseVerletLists"},
+            {ContainerOption::octree, "Octree"},
+            {ContainerOption::hierarchicalGrids, "HierarchicalGrids"}};
```

Comment:
```suggestion
            {ContainerOption::hierarchicalGrids, "HierarchicalGrids"},
            };
```
This trailing comma should restore the formatting by clang-format.

Reply:
Done